test/HealthChecks.Uris.Tests/Functional/UrisHealthCheckTests.cs

```diff
@@ -234,5 +234,69 @@ public async Task be_healthy_if_request_sucess_and_timeout_is_configured()
             response.StatusCode
                     .Should().Be(HttpStatusCode.OK);
         }
+
+        [Fact]
+        public async Task be_healthy_if_request_succeeds_and_expected_response_matches()
+        {
+            var uri = new Uri("https://httpbin.org/robots.txt");
+
+            var webHostBuilder = new WebHostBuilder()
+                .ConfigureServices(services =>
+                {
+                    services.AddHealthChecks()
+                        .AddUrlGroup(opt =>
+                        {
+                            opt.AddUri(uri);
+                            opt.ExpectContent("User-agent: *");

```

Comment:
GET request to https://httpbin.org/robots.txt returns 
```
User-agent: *
Disallow: /deny
```

Reply:
I think you missed the right place to configure. See `UriHealthCheckOptions.AddUri` and remove `UriHealthCheckOptions.ExpectedContent` from "root" options class.

Reply:
> I think you missed the right place to configure. See `UriHealthCheckOptions.AddUri` and remove `UriHealthCheckOptions.ExpectedContent` from "root" options class.

That indeed was an misconfiguration I made. But it was resolved with my latest commit

Reply:
@sungam3r have a look at this commit;

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/1199/commits/4f37886defce1324ec58f9a505440e52483c0a0b#diff-2dcf97f84b61d62b41d267476b11e16528bc47af8713e8d90352c82397090117

Reply:
https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/1199/files#r927400716